### DISCUSSION TITLE
Stat: Ability to manually set unit font size

### DISCUSSION THREAD

**Discussion by RyuunosukeDS3** [grafana/grafana#38285] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
The ability to manually set the font size of units on Stat and gauge panel.

**Why is this needed**:
While the unit scaling might seem ok for general purpose, in my company we use grafana dashboards on big tvs hanging on the wall. So, ppl would appreciate to see the number from far away. Since we have way too much info on the screen making the unit smaller in order to make the number bigger was vital to our work environment. With that said, I actually managed to make the units smaller with some CSS trickery using the Boom theme Panel.
Here is the code to do it:

```
.panel-content.panel-content--no-padding > div > div > div > div > div > div > span:nth- 
child(2) {
     font-size: 15px !important;
}
```

But it would be really nice to have a field where we can manually set it natively.

Example of smaller unit:
![image](https://user-images.githubusercontent.com/31015675/119346931-34e9dc80-bc71-11eb-95d9-bf9bb3ca9389.png)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
